packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -44,85 +80,134 @@ const storybookMeta: Meta<typeof Autocomplete> = {
   args: {
     label: "Destination",
     hint: "Select your destination in the Sol system.",
+    options: stations,
   },
   decorators: [MuiThemeDecorator],
 };
 
 export default storybookMeta;
 
-type StationType = { label: string };
-
-const stations: ReadonlyArray<StationType> = [
-  { label: "Anderson Station" },
-  { label: "Bara Gaon Complex" },
-  { label: "Ceres" },
-  { label: "Corley Station" },
-  { label: "Deep Transfer Station Three" },
-  { label: "Eros" },
-  { label: "Free Navy Supply Depot" },
-  { label: "Ganymede" },
-  { label: "Gewitter Base" },
-  { label: "Iapetus Station" },
-  { label: "Kelso Station" },
-  { label: "Laconian Transfer Station" },
-  { label: "Mao Station" },
-  { label: "Medina Station" },
-  { label: "Nauvoo" },
-  { label: "Oshima" },
-  { label: "Osiris Station" },
-  { label: "Pallas" },
-  { label: "Phoebe Station" },
-  { label: "Prospero Station" },
-  { label: "Shirazi-Ma Complex" },
-  { label: "Terryon Lock" },
-  { label: "Thoth Station" },
-  { label: "Tycho Station" },
-  { label: "Vesta" },
-];
-
 type AutocompleteType = AutocompleteProps<
   StationType | undefined,
   boolean | undefined,
   boolean | undefined
 >;
 
-export const Default: StoryObj<AutocompleteType> = {
+const Template: StoryObj<AutocompleteType> = {
   render: function C(props) {
-    return <Autocomplete {...props} options={stations} />;
+    return <Autocomplete {...props} />;
+  },
+};
+
+export const Default: StoryObj<AutocompleteType> = {
+  ...Template,
+  play: async ({ canvasElement, step }) => {
+    await step("Filter and Select from listbox", async () => {
+      const canvas = within(canvasElement);
+      const comboBox = canvas.getByRole("combobox") as HTMLInputElement;
+      userEvent.click(comboBox);
+      const listbox = screen.getByRole("listbox");
+      expect(listbox).not.toBeNull();
```

Comment:
Can we also name this `listboxElement`? That way, it's easier to tell "oh, this is a DOM element". Later in the code, I was confused what `noOpts` was, now I know it's an element.